src/css.js

```diff
@@ -58,13 +58,31 @@ function vendorPropName( name ) {
 	}
 }
 
+// Use font-size to convert to px
+function convertToPx( elem, value ) {
+	var fontSize = jQuery.css( elem, "font-size" );
+	elem.style.fontSize = value;
+	value = parseFloat( jQuery.css( elem, "font-size" ) );
+	elem.style.fontSize = fontSize;
```

Comment:
This function has some problems:
* It doesn't work when `value` is expressed with `%`, because `height`/`width` and `font-size` have different referents
* It leaves an undesirable inline `font-size`
* (nit) It invokes `parseFloat` directly rather than relying on `jQuery.css( elem, prop, "" )` for coercion